bin/node/runtime/src/lib.rs

```diff
@@ -1066,6 +1128,9 @@ impl pallet_vesting::Config for Runtime {
 	type BlockNumberToBalance = ConvertInto;
 	type MinVestedTransfer = MinVestedTransfer;
 	type WeightInfo = pallet_vesting::weights::SubstrateWeight<Runtime>;
+	// `VestingInfo` encode length is 36bytes. 28 schedules gets encoded as 1009 bytes, which is the
+	// highest number of schedules that encodes less than 2^10.
+	const MAX_VESTING_SCHEDULES: u32 = 28;
```

Comment:
[Support multiple vesting schedules per account. ](https://github.com/paritytech/substrate/pull/9202)
[The issue it solves](https://github.com/paritytech/substrate/issues/7101)